docs/standard/io/media/pipelines/do-not-use-include1.md

```diff
@@ -0,0 +1,2 @@
+> [!WARNING]
```

Comment:
Includes are supposed to go here: https://github.com/dotnet/docs/tree/master/includes

Reply:
I had them there and they didn't resolve. I'll try again.

Reply:
Do absolute links, ie, `(~/includes/myInclude.md)` work? I couldn't get that working.

Reply:
they should, but we use relative paths for includes so folks editing the article can easily find them.

Reply:
absolute are much easier to reason about. They can be copy/pasted anywhere. We've had to move a few folders, and absolute links don't break.